src/NuGet.Server.V2/OData/Serializers/NuGetEntityTypeSerializer.cs

```diff
@@ -45,6 +46,10 @@ private void TryAnnotateV2FeedPackage(ODataEntry entry, EntityInstanceContext en
                 {
                     atomEntryMetadata.Updated = instance.LastUpdated;
                 }
+                if (instance.Published > DateTime.MinValue)
+                {
+                    atomEntryMetadata.Published = instance.Published;
+                }
```

Comment:
This is enriching the Atom published property for each package with the package's published date. This is analogous to what we already do with the updated Atom property.